utilities/blob_db/blob_db_impl.cc

```diff
@@ -1523,22 +1521,8 @@ std::pair<bool, int64_t> BlobDBImpl::EvictCompacted(bool aborted) {
 
   override_packet_t packet;
   while (override_vals_q_.dequeue(&packet)) {
-    bool succ = FindFileAndEvictABlob(packet.file_number_, packet.key_size_,
-                                      packet.blob_offset_, packet.blob_size_);
-
-    if (!succ)
-      ROCKS_LOG_DEBUG(
-          db_options_.info_log,
-          "EVICT COMPACTION FAILURE SN: %d FN: %d OFFSET: %d SIZE: %d",
-          packet.dsn_, packet.file_number_, packet.blob_offset_,
-          packet.blob_size_);
-
-    if (debug_level_ >= 3)
-      ROCKS_LOG_INFO(
-          db_options_.info_log,
-          "EVICT COMPACTED SN: %d FN: %d OFFSET: %d SIZE: %d SUCC: %d",
-          packet.dsn_, packet.file_number_, packet.blob_offset_,
-          packet.blob_size_, succ);
+    FindFileAndEvictABlob(packet.file_number_, packet.key_size_,
+                          packet.blob_offset_, packet.blob_size_);
```

Comment:
I looked at the code in `FindFileAndEvictABlob` to see in what case it returns false, and as you said, it seems to be only in the case where the file is already deleted. So, the code as is, is fine with me.